aesara/link/numba/dispatch/scalar.py

```diff
@@ -241,3 +242,15 @@ def inv(x):
         return 1 / x
 
     return inv
+
+
+# Override the more general ScalarOp implementation
+# to avoid using scipy.special in numba and make the
+# implementation transparent for numba
```

Comment:
```suggestion
```
This doesn't really need an explanation, especially since the `numba-scipy` logic is only a fallback for missing `ScalarOp` implementations.